src/group_theory/group_action/defs.lean

```diff
@@ -46,16 +46,6 @@ variables {M N G A B α β γ : Type*}
 
```

Comment:
The docstring for this file should say that `has_scalar` is defined elsewhere, rather than claiming it is defined here.